mitmproxy/filt.py

```diff
@@ -261,6 +326,7 @@ class FSrc(_Rex):
     help = "Match source address"
     is_binary = False
 
+    @only(HTTPFlow, TCPFlow)
```

Comment:
Same over here. We currently only have HTTPFlow and TCPFlow, but `.client_conn` is already an attribute of `Flow`, so this should extend to all flows we'll ever encounter. Let's just strip off the decorator.